answers/advanced/promisify-a.js

```diff
@@ -9,3 +9,34 @@ function promisify(fn) {
         });
     }
 }
+
+/*
+function promisify(fn) {
+    return function (...args) {
+        return new Promise(function (resolve, reject) {
+            function cb(result) {
+                resolve(result);
+            }
+
+            fn.apply(this, [cb].concat(args));
```

Comment:
Thanks for adding this?

Mind changing this to match line 8?

Reply:
Thanks! Updated code to match line 8.